packages/material-ui/src/Chip/Chip.js

```diff
@@ -38,182 +115,211 @@ export const styles = (theme) => {
+          marginRight: 3,
+        },
+      }),
+      /* Styles applied to the root element if `variant="outlined"` and `color="primary"`. */
+      ...(styleProps.variant === 'outlined' &&
+        styleProps.color === 'primary' && {
+          color: theme.palette.primary.main,
+          border: `1px solid ${theme.palette.primary.main}`,
+          [`&.Mui-focusVisible, .${chipClasses.clickable}&:hover`]: {
+            backgroundColor: alpha(theme.palette.primary.main, theme.palette.action.hoverOpacity),
+          },
+        }),
+      /* Styles applied to the root element if `variant="outlined"` and `color="secondary"`. */
+      ...(styleProps.variant === 'outlined' &&
+        styleProps.color === 'secondary' && {
+          color: theme.palette.secondary.main,
+          border: `1px solid ${theme.palette.secondary.main}`,
+          [`&.Mui-focusVisible, .${chipClasses.clickable}&:hover`]: {
+            backgroundColor: alpha(theme.palette.secondary.main, theme.palette.action.hoverOpacity),
+          },
+        }),
+    };
+  },
+  ({ theme, styleProps }) => {
+    const backgroundColor =
+      theme.palette.mode === 'light' ? theme.palette.grey[300] : theme.palette.grey[700];
+
+    return {
+      /* Styles applied to the root element if `onClick` is defined or `clickable={true}`. */
+      ...(styleProps.clickable && {
+        userSelect: 'none',
+        WebkitTapHighlightColor: 'transparent',
+        cursor: 'pointer',
+        '&:hover, &.Mui-focusVisible': {
+          backgroundColor: emphasize(backgroundColor, 0.08),
+        },
+        '&:active': {
+          boxShadow: theme.shadows[1],
+        },
+      }),
+      /* Styles applied to the root element if `onClick` and `color="primary"` is defined or `clickable={true}`. */
+      ...(styleProps.clickable &&
+        styleProps.color === 'primary' && {
+          '&:hover, &.Mui-focusVisible': {
+            backgroundColor: emphasize(theme.palette.primary.main, 0.08),
+          },
+        }),
+      /* Styles applied to the root element if `onClick` and `color="secondary"` is defined or `clickable={true}`. */
+      ...(styleProps.clickable &&
+        styleProps.color === 'secondary' && {
+          '&:hover, &.Mui-focusVisible': {
+            backgroundColor: emphasize(theme.palette.secondary.main, 0.08),
+          },
+        }),
+    };
+  },
+);
+
+const ChipLabel = experimentalStyled(
+  'span',
+  {},
+  {
+    name: 'MuiChip',
+    slot: 'Label',
+  },
+)(({ styleProps }) => ({
+  /* Styles applied to the label `span` element. */
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
+  paddingLeft: 12,
+  paddingRight: 12,
+  whiteSpace: 'nowrap',
+  /* Styles applied to the label `span` element if `size="small"`. */
+  ...(styleProps.size === 'small' && {
+    paddingLeft: 8,
+    paddingRight: 8,
+  }),
+}));
+
+const ChipIcon = experimentalStyled(
+  'span',
+  {},
+  {
+    name: 'MuiChip',
+    slot: 'Icon',
+  },
+)(({ theme, styleProps }) => ({
```

Comment:
Move these styles in the `root`, by using class names selectors similar to the `avatar`